Mathlib/GroupTheory/GroupAction/Basic.lean

```diff
@@ -212,6 +212,66 @@ end Stabilizers
 
 end MulAction
 
+namespace MulDistribMulAction
+
+variable (M : Type u) [Monoid M] (α : Type v)
+
+/-- The submonoid of elements fixed under the whole action. -/
+def fixedSubmonoid [Monoid α] [MulDistribMulAction M α] : Submonoid α where
```

Comment:
Note that these names are at odds with [FixedPoints.subfield](https://leanprover-community.github.io/mathlib4_docs/Mathlib/FieldTheory/Fixed.html#FixedPoints.subfield), [FixedPoints.intermediateField](https://leanprover-community.github.io/mathlib4_docs/Mathlib/FieldTheory/Galois.html#FixedPoints.intermediateField), etc.